.github/workflows/debos.yml

```diff
@@ -115,7 +118,8 @@ jobs:
           dir="${FILESERVER_DIR}/${id}"
           mkdir -vp "${dir}"
           # copy output files
-          cp -v disk.img "${dir}"
+          cp -v disk-4096.img.gz "${dir}"
```

Comment:
I don't find the names disk-512.img and disk-4096.img to be really good.. but I cannot find a better name for now..